templates/zerver/app/index.html

```diff
@@ -274,6 +274,19 @@
                         <div id="page_loading_indicator"></div>
                         <div id="message_feed_errors_container"></div>
                         <div id="message-lists-container"></div>
+                        <div id="message-list-navigation">
+                            <div id="message-list-navigation-buttons-wrapper">
+                                <button id="message-list-navigation-mark-as-read" class="message-list-navigation-button">
+                                    {{ _('Mark as read') }}
+                                </button>
+                                <button id="message-list-navigation-next-unread-conversation" class="message-list-navigation-button">
+                                    {{ _('Next unread conversation') }}
+                                </button>
+                                <button id="message-list-navigation-home-view" class="message-list-navigation-button" data-tippy-placement="bottom-start">
+                                    {{ _('Home view') }}
+                                </button>
+                            </div>
+                        </div>
```

Comment:
Probably we'll want the body of this to be in a Handlebars template, with just the outer element here.

Reply:
Extracted them to `message-list-navigation-buttons.html`.